azure/azevents/azevents.go

```diff
@@ -327,10 +327,13 @@ func (s *Server) handleParsedPortfolio(id string, resp *task.ParsePortfolioRespo
 			if err != nil {
 				return fmt.Errorf("creating blob %d: %w", i, err)
 			}
+
+			// TODO: There's other metadata in output.Portfolio, like `InvestorName`, that
```

Comment:
Can you add a bug for this? I don't want to lose track of it.

Reply:
Done